elisp/intero.el

```diff
@@ -1455,6 +1495,11 @@ Completions for PREFIX are passed to CONT in SOURCE-BUFFER."
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Process communication
 
+(defun intero-call-process (program &optional infile destination display &rest args)
+  "Synchronously call a process. Same interface as call-process/process-file."
+  (let ((process-args (append (list program infile destination display) args)))
+    (apply 'process-file process-args)))
```

Comment:
Exactly. The docstring is really lacking in that main point! I'll expand that in future work.